src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,12 +1,36 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { User } from '../../types/User';
+import { getUser } from '../../api';
+
+type Props = {
+  todo: Todo;
+  onClose: () => void;
+};
+
+export const TodoModal: React.FC<Props> = props => {
+  const { todo, onClose } = props;
+
+  const [isloading, setIsLoading] = useState(true);
```

Comment:
The variable `isloading` should be renamed to `isLoading` to follow camelCase naming conventions, which is a common practice in JavaScript and TypeScript.